nntrainer/layers/pow_layer.cpp

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 SeungBaek Hong <sb92.hong@samsung.com>
+ *
+ * @file   pow_layer.cpp
+ * @date   20 Nov 2024
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author SeungBaek Hong <sb92.hong@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  This is pow layer class (operation layer)
+ *
+ */
+
+#include "common_properties.h"
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <node_exporter.h>
+#include <pow_layer.h>
+#include <util_func.h>
+
+#include <layer_context.h>
+
+namespace nntrainer {
+
+void PowLayer::finalize(InitLayerContext &context) {
+  context.setOutputDimensions({context.getInputDimensions()[0]});
+}
+
+void PowLayer::forwarding_operation(const Tensor &input, Tensor &hidden) {
+  float exp = std::get<props::Exponent>(pow_props).get();
```

Comment:
Can we change the name `exp` ? (exp function is defined in <cmath>)

Reply:
I've modified it(`exp` -> `exponent`). Thanks a lot!